_data/glossary.yml

```diff
@@ -0,0 +1,31 @@
+canonical-tags:
+- Fundamentals
+- K8s Objects
```

Comment:
Also, I think that the canonical references here should be case sensitive, but when we do the automated checks, that should be case insensitive to avoid some headaches.